lib/devise/token_authenticatable/model.rb

```diff
@@ -36,7 +36,13 @@ module Models
     #
     module TokenAuthenticatable
       extend ActiveSupport::Concern
-
+      
+      Devise::Models.config(self, :should_reset_authentication_token, :should_ensure_authentication_token)
```

Comment:
I also would appreciate a renaming of these two configurations to `reset_authentication_token?` and `ensure_authentication_token?` or similar.